<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Amazonski pragozd: Mila Stimec</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="scoreTE"/>
      </item>
      <item>
       <widget class="QLineEdit" name="uporabnikLE">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Uporabniski ime</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="replayPB">
        <property name="text">
         <string>Replay</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closePB">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newGamePB">
        <property name="text">
         <string>New Game</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadPB">
        <property name="text">
         <string>Load Game</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
